src/viewport.js

```diff
@@ -988,6 +1007,34 @@ $.Viewport.prototype = {
             this.degreesSpring.target.value;
     },
 
+    /**
+     * Rotates this viewport to the angle specified. Alias for setRotation to be consistent with with pan and zoom API
+     * @function
+     * @param {Number} degrees The degrees to set the rotation to.
+     * @param {Boolean} [immediately=false] Whether to animate to the new angle
+     * or rotate immediately.
+     * @param {OpenSeadragon.Point} [pivot=false] (Optional) point in viewport coordinates
+     * around which the rotation should be performed. Defaults to the center of the viewport.
+     * @returns {OpenSeadragon.Viewport} Chainable.
+     */
+    rotateTo: function(degrees, immediately, pivot){
```

Comment:
Seems like a nice idea to add these aliases. One way it's different from `panTo` and `zoomTo` is that `zoomTo` takes in the zoom value, then the pivot point, and then the `immediately`, whereas with this function the last two are flipped. If we change it to align with `zoomTo`, then it's inconsistent with `setRotation` (which we shouldn't change, because it'll break existing apps). So, which thing is it more important to be consistent with? Possibly `zoomTo`. 

Of course, if we do that, people will have to call `rotateTo` with an explicit `null` for the pivot if they want to pass `true` for the `immediately`. It's a similar situation if we don't flip them and you want the pivot without specifying `immediately`, so I think that's a toss up.

Reply:
I suppose one approach to the inconsistency is maybe we don't add `pivot` to `setRotation` at all, and instead move all of its code into `rotateTo` and have `setRotation` call `rotateTo`. In that way, we would be moving toward `rotateTo` as the primary API here. 

That said, we'll want to continue with the `setRotation` terminology in `TiledImage`, since its API is all get/set, so it's probably just as well to support `pivot` in the `setRotation` here in `Viewport`. 

Just thinking out loud here!

Reply:
Good point! I'm updating the order of the arguments in `rotateTo` and `rotateBy` to match the order in `zoomTo/zoomBy`. Since these are new methods, it can't break existing code, and it is more consistent. I like your idea of making the original  `setRotation` the almost-alias, and moving the bulk of the code into `rotateTo` which is more consistent with pan/zoom API.